src/main/java/org/cactoos/scalar/Repeated.java

```diff
@@ -48,23 +46,23 @@
     /**
      * Ctor.
      *
-     * <p>If {@code max} is equal or less than zero {@link #call()} will return
+     * <p>If {@code max} is equal or less than zero {@link #value()} will
+     * return
      * an exception.</p>
      *
-     * @param cllbl Callable to repeat.
+     * @param sclr Scalar to repeat.
      * @param count How many times.
      */
-    public RepeatedCallable(final Callable<X> cllbl, final int count) {
-        this.callable = cllbl;
+    public Repeated(final Scalar<T> sclr, final int count) {
+        this.scalar = sclr;
         this.times = count;
     }
 
     @Override
-    public X call() throws Exception {
-        return new Repeated<>(
-            new FuncOf<>(this.callable),
+    public T value() throws Exception {
+        return new org.cactoos.func.Repeated<>(
```

Comment:
@fabriciofx Can this be moved to the primary ctor?

Reply:
@andreoss done!